twin/README.md

```diff
@@ -0,0 +1,5 @@
+# This Week in Nushell
+
+[RSS](/rss.xml) / [Atom](/atom.xml)
```

Comment:
What's your intention and suggestion regarding feeds? The feed plugin config was not adjusted so would not include twin posts, but this twin index still links to the feeds.

If we split twin from feeds, I would expect separate feeds too.